modules/model/shared/src/main/scala/gem/data/EnumZipper.scala

```diff
@@ -1,13 +1,14 @@
 // Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
-package gpp.util
+package gem
+package data
 
 import cats.kernel.Eq
 import cats.implicits._
 import gem.util.Enumerated
 
-class EnumZipper[A](lefts: List[A], focus: A, rights: List[A])
+class EnumZipper[A] protected (lefts: List[A], focus: A, rights: List[A])
```

Comment:
Isn't this private in `Zipper`?

Reply:
Sorry, lost that in the refactor. I added it now, but it's `protected` in `Zipper` so that it can be constructed from `EnumZipper` subclass. `EnumZipper`'s constructor is now `private`.